sync3/caches/user.go

```diff
@@ -164,17 +164,16 @@ func (i *InviteData) RoomMetadata() *internal.RoomMetadata {
 	if i.RoomType != "" {
 		roomType = &i.RoomType
 	}
-	return &internal.RoomMetadata{
```

Comment:
Was it a problem that we didn't make a ChildSpaceRooms map here, too? Ditto for the two hunks below.

Reply:
Nope as they were stubs and not used by the global cache